src/scanners/javascript.js

```diff
@@ -71,6 +71,9 @@ export default class JavaScriptScanner {
       // Avoid loading the addons-linter .eslintrc file
       useEslintrc: false,
 
+      // Avoid loading the .eslintignore file
+      ignore: false,
```

Comment:
The option seems to be the right one, but based on the current test failures and a quick look to the eslint sources it seems that setting ignore to false does have the side effect of making eslint to ignore baseConfig.ignorePatterns and to only use the default ones  (which exclude dot files and node_modules, that are exactly the two patterns that we do want to include with a negated ignore pattern instead).

I need to take another look to see if there is any way to fix it on our side (or if the only option would be to open an upstream issue in the eslint repository).